be/src/olap/rowset/alpha_rowset_reader.h

```diff
@@ -103,6 +128,9 @@ class AlphaRowsetReader : public RowsetReader {
     RowsetReaderContext* _current_read_context;
     OlapReaderStatistics _owned_stats;
     OlapReaderStatistics* _stats = &_owned_stats;
+
+    // a priority queue for merging rowsets
+    std::priority_queue<RowCursorWithOrdinal, vector<RowCursorWithOrdinal>, RowCursorWithOrdinalComparator> _merge_queue;
```

Comment:
why not use use **RowCursorWithOrdinal** as element of queue. If then, there is no need to record ordinal.

Reply:
I removed the RowCursorWithOrdinal, and use MergeContext directly